packages/@coorpacademy-components/src/molecule/learner-skill-card/index.js

```diff
@@ -0,0 +1,180 @@
+import React, {useCallback, useState} from 'react';
+import {convert} from 'css-color-function';
+import {get, getOr} from 'lodash/fp';
+import PropTypes from 'prop-types';
+import Icon from '../../atom/icon';
+import ButtonLink from '../../atom/button-link';
+import Provider from '../../atom/provider';
+import style from './style.css';
+
+const LearnerSkillCard = (props, context) => {
+  const {
+    'aria-label': ariaLabel,
+    skillTitle,
+    skillAriaLabel,
+    metrics,
+    onReviewClick,
+    onExploreClick
+  } = props;
+  const {skillCourses, skillQuestions, completedCourses = 0} = metrics;
+  const {skin, translate} = context;
+  const [hovered, setHovered] = useState(false);
+  const primarySkinColor = getOr('#0061FF', 'common.primary', skin);
+
+  const review = translate('Review');
+  const explore = translate('Explore');
+  const courses = translate('courses');
+  const questions = translate('questions');
+  const skillFocus = translate('skill_focus');
+  const coursesCompleted = translate('courses_completed');
+
+  const handleMouseOver = useCallback(() => setHovered(true), [setHovered]);
+
+  const handleMouseLeave = useCallback(() => setHovered(false), [setHovered]);
+
+  const buttonReviewProps = {
+    customStyle: {
+      backgroundColor: '#FFF',
+      transition: 'background-color 0.15s ease-in-out, color 0.15s ease-in-out'
+    },
+    onReviewClick,
+    'aria-label': `${skillTitle}, ${review}`,
+    label: review,
+    'data-name': 'learner-skill-card-review-button'
+  };
+
+  const buttonExploreProps = {
+    customStyle: {
+      backgroundColor: hovered ? primarySkinColor : convert(`color(${primarySkinColor} a(0.07))`),
+      color: hovered ? '#FFFFFF' : primarySkinColor,
+      transition: 'background-color 0.15s ease-in-out, color 0.15s ease-in-out'
+    },
+    onExploreClick,
+    'aria-label': `${skillTitle}, ${explore}`,
+    label: explore,
+    'data-name': 'learner-skill-card-explore-button',
+    icon: {
+      position: 'left',
+      faIcon: {
+        name: 'compass',
+        backgroundColor: hovered ? primarySkinColor : convert(`color(${primarySkinColor} a(0.07))`),
+        color: hovered ? '#FFFFFF' : primarySkinColor,
+        size: 16
+      }
+    }
+  };
+
+  const completedPercentage =
+    skillCourses && Number.parseInt((completedCourses / skillCourses) * 100);
+
+  const ProgressBar = useCallback(() => {
+    const progressBarColor = '#3EC483';
+    const inlineProgressValueStyle = {
+      backgroundColor: progressBarColor,
+      width: `${completedPercentage}%`
+    };
+
+    if (!skillCourses) {
+      return null;
+    }
```

Comment:
you can exit earlier, and one line it  `if (!skillCourses) return null`.

Or even set the ProgressBar itself to null.  (I'm unsure about that part, I'm rusty with react component lifecycle)